ephios/plugins/basesignup/signup/section_based.py

```diff
@@ -295,13 +295,9 @@ def _configure_participation(
         participation.state = AbstractParticipation.States.REQUESTED
         return participation
 
-    def render_configuration_form(self, *args, form=None, **kwargs):
+    def get_configuration_form_template(self):
         """We overwrite the template to render the formset."""
-        form = form or self.get_configuration_form(*args, event=self.event, **kwargs)
-        template = get_template("basesignup/section_based/configuration_form.html").render(
-            {"form": form}
-        )
-        return template
+        return get_template("basesignup/section_based/configuration_form.html")
```

Comment:
I wonder if we could be more close to how the django class based views work and just provide the class attribute `configuration_form_template_name` and just load from that name in `render_configuration_form`. That way you wouldn't need to overwrite a method, just the attribute - and you could reuse a template containing just

```jinja2
{% load crispy_forms_filters %}
{{ form|crispy }}
```

in both the base signup method and beachguard without having to repeat the basic implementation (as seen in https://github.com/ephios-dev/ephios/pull/990/files) or doing some complicated `super(SectionBasedSignupMethod, ...)` over there.

After all, signup methods are complicated already, so I think using established django petterns would fit here.

Reply:
Maybe Django's new form Templates are also worth looking into...

Reply:
I think looking into form templates is something for later. I refactored to use an attribute, please re-review